packages/core/src/domain/tracekit/computeStackTrace.ts

```diff
@@ -1,556 +1,128 @@
-import type { StackFrame, StackTrace } from './types'
+import type { StackTrace, StackFrame } from './types'
 
 const UNKNOWN_FUNCTION = '?'
 
 /**
  * computeStackTrace: cross-browser stack traces in JavaScript
- *
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about removing this comment too?